### PR TITLE
chore(ci): Lower glob_minimum_cooldown_ms for k8s e2e tests

### LINKED ISSUE #8009: Lower `glob_minimum_cooldown` to 5s for k8s e2e tests
I recently merged #7980 to put the default back to the released value in 0.14.0 until we can take a look in #7840 however this results in the tests taking 5 hours rather than 15 minutes so it'd be nice to drop the cooldown just for the tests at least until we can evaluate and settle on a new default.

cc/ @spencergilbert @StephenWakely

### PR DESCRIPTION
**Pull request by jszwedko**
In #7980 we put the default back to 60000 ms until we can take a look in
 #7840 however this results in the tests taking 5 hours rather than 15
minutes so this drops the cooldown just for the tests.

Closes #8009 

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

**Review by spencergilbert — COMMENTED** [review]
Looks legit... though thinking about it now, I'm not sure we can properly merge the contents of the `rawConfig`... I haven't tried and happy to be corrected

**Review by spencergilbert — APPROVED** [review]
🤞 tests pass

File: lib/k8s-test-framework/src/vector.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ pub struct CommandBuilder {
     interface_command: String,
     namespace: String,
     helm_chart: String,
-    custom_helm_values_file: Option<HelmValuesFile>,
+    custom_helm_values_files: Vec<HelmValuesFile>,
```
Comment (StephenWakely) [inline lib/k8s-test-framework/src/vector.rs:13]: Nice! This is much better.

**Review by StephenWakely — COMMENTED** [review]
Beautiful.

**Pull request merged**